scripts/pages.js

```diff
@@ -30,6 +30,8 @@ module.exports = [
     ['resources',                          'app/resources/index',                       'default',    'Resources'],
     ['resources/asset_indexws',            'app/resources/asset_index',                 'default',    'Asset Index', 'ja'],
     ['resources/market_timesws',           'app/resources/trading_times',               'default',    'Trading Times'],
+    ['resources/economic_calendar',        'app/resources/economic_calendar',           'default',    'Economic Calendar'],
```

Comment:
do we need to exclude it for japan?

Reply:
@ashkanx , do we need to? Since Trading Times isn't excluded I assume it's the same with Economic Calendar.